src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/AcceptOrRejectOfferTest.java

```diff
@@ -57,7 +57,7 @@ public void beforeEachTest() throws Exception {
         );
 
         claim = claimStore.saveClaim(SampleClaimData.builder().build(), SUBMITTER_ID, LocalDate.now());
-        claimRepository.linkDefendant(claim.getId(), DEFENDANT_ID);
+        caseRepository.linkDefendant(claim.getExternalId(), "2", BEARER_TOKEN);
```

Comment:
Reusing `DEFENDANT_ID` is more readable. As a reader I am surprised that some random defendant identified by ID equal 2 is used.

Reply:
accidental 👍